api/proto/teleport/workloadidentity/v1/attrs.proto

```diff
@@ -76,6 +76,24 @@ message WorkloadAttrsPodmanPod {
   map<string, string> labels = 2;
 }
 
+// Attributes sourced from the Docker workload attestor.
+message WorkloadAttrsDocker {
+  // Whether the workload passed Docker attestation.
+  bool attested = 1;
+  // Attributes of the container.
+  WorkloadAttrsDockerContainer container = 2;
+}
+
+// Attributes of the container sourced from the Docker workload attestation.
+message WorkloadAttrsDockerContainer {
```

Comment:
I'm split on whether we should move these fields up to the WorkloadAttrsDocker message, or, whether it's more convenient to have consistency with the Podman attestor here.

Reply:
Yeah me too! I started out with them on the root message, but then moved them (A) for consistency and (B) because we might want to include things like the Docker Engine version on the root object in the future.